test/spec/algoliasearch.helper/derive/detach.js

```diff
@@ -20,34 +18,14 @@ test('[derived helper] detach a derived helper', function(t) {
   function searchTest(requests) {
     nbRequest = nbRequest || 0;
     if (nbRequest === 0) {
-      t.equal(
-        requests.length,
-        2,
-        'the helper generates two queries'
-      );
-      t.deepEqual(
-        requests[0],
-        requests[1],
-        'the helper generates the same query twice'
-      );
-      t.equal(
-        derivedHelper.listeners('result').length,
-        1,
-        'one listener is plugged to the derived helper'
-      );
+      expect(requests.length).toBe(2);
+      expect(requests[0]).toEqual(requests[1]);
+      expect(derivedHelper.listeners('result').length).toBe(1);
```

Comment:
We can use [`toHaveLength(n)`](https://jestjs.io/docs/en/expect#tohavelengthnumber).

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453